lib/node_modules/@stdlib/math/base/special/factorial2/lib/main.js

```diff
@@ -63,7 +59,7 @@ function factorial2( n ) {
 	if ( isnan( n ) ) {
 		return NaN;
 	}
-	if ( n >= MAX_FACTORIAL2 ) {
+	if ( n >= FLOAT64_MAX_SAFE_NTH_DOUBLE_FACTORIAL ) {
```

Comment:
```suggestion
	if ( n > FLOAT64_MAX_SAFE_NTH_DOUBLE_FACTORIAL ) {
```

Which would also mean changing this, after the constant has been corrected.